<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMemo</class>
 <widget class="QMainWindow" name="MyMemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>510</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MyMemo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="_src.qrc">
    <normaloff>:/new/prefix1/gp_source/icon.png</normaloff>:/new/prefix1/gp_source/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>205</y>
      <width>130</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;刪除事項&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="_src.qrc">
      <normaloff>:/new/prefix1/gp_source/newRemove.png</normaloff>:/new/prefix1/gp_source/newRemove.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>205</y>
      <width>130</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;列出我的事項&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="_src.qrc">
      <normaloff>:/new/prefix1/gp_source/list.png</normaloff>:/new/prefix1/gp_source/list.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>205</y>
      <width>130</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;儲存更新並結束&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="_src.qrc">
      <normaloff>:/new/prefix1/gp_source/renew.png</normaloff>:/new/prefix1/gp_source/renew.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>-10</y>
      <width>931</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/gp_source/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/gp_source/board.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>135</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lsAll">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>132</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>205</y>
      <width>130</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;記錄事項&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="_src.qrc">
      <normaloff>:/new/prefix1/gp_source/newAdd.png</normaloff>:/new/prefix1/gp_source/newAdd.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>831</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 161, 241, 40%);
color: rgb(97, 97, 97, 80%);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="saveAndLeave">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>125</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>205</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>205</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>205</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="proBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>467</y>
      <width>130</width>
      <height>6</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>216</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>216</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;清松手寫體2&quot;;
color: rgb(83, 83, 83);</string>
    </property>
    <property name="text">
     <string>release ver. 3.1 202007261446</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>205</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>205</y>
      <width>130</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>設定</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="_src.qrc">
      <normaloff>:/new/prefix1/gp_source/settings.png</normaloff>:/new/prefix1/gp_source/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>74</width>
      <height>74</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="settings">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>126</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;清松手寫體2&quot;;
color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>✿新增</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;清松手寫體2&quot;;
color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>✿刪除</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;清松手寫體2&quot;;
color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>✿清單</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;清松手寫體2&quot;;
color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>✿設定</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;清松手寫體2&quot;;
color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>✿儲存</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>lsAll</zorder>
   <zorder>label_3</zorder>
   <zorder>label_8</zorder>
   <zorder>saveAndLeave</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>proBar</zorder>
   <zorder>label_9</zorder>
   <zorder>line_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>settings</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="_src.qrc"/>
 </resources>
 <connections/>
</ui>
